src/main/groovy/com/avast/gradle/dockercompose/tasks/ComposeDown.groovy

```diff
@@ -23,7 +23,7 @@ class ComposeDown extends DefaultTask {
             if (extension.removeContainers) {
                 if (getDockerComposeVersion() >= VersionNumber.parse('1.6.0')) {
                     project.exec { ExecSpec e ->
-                        e.commandLine extension.composeCommand('down')
+                        e.commandLine extension.composeCommand('down', '--rmi', 'local', '--volumes')

```

Comment:
hmm I understood it differently, I thought that it is images that are built but tagged as a a specified imagename

If you specify image as well as build, then Compose tags the built image with the tag specified in image